gitignore.go

```diff
@@ -65,6 +66,9 @@ func (g gitIgnore) Match(path string, isDir bool) bool {
 	if err != nil {
 		return false
 	}
+	if runtime.GOOS == "windows" {
```

Comment:
In Go way, We prefer OS independent code.
I propose that you use [filepath.ToSlash](https://golang.org/pkg/path/filepath/#ToSlash) which dosen't use runtime package.

Reply:
Oh, thanks! I didn't know `filePath.ToSlash` existed. I pushed another implementing this suggestion :)

Reply:
Thanks!